wiki/en/Installation-for-Linux.md

```diff
@@ -54,22 +54,43 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If no instance of a JACK server is running, one is normally started automatically.
+
+Jamulus will attach to a running JACK instance by name, using the `JACK_DEFAULT_SERVER` environment variable or `default` if the variable is not set.
+
+If your distribution uses JACK as the sound server, check out [how to configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl).
+
+If your distribution uses PipeWire by default, check out the section on [configuring PipeWire](Installation-for-Linux#configure-pipewire).
+
 ### Configure JACK with QjackCtl
 
 Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
 
-1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**
+1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**.
 2. Configure your audio hardware as follows (the exact settings for JACK will depend on what your audio hardware is capable of):
 
-- Set the audio **Interface** to the one you want (there may be several in the list)
-- Set the **Sample Rate to 48000**
-- Set the **Frames/Period to 128** and Periods/Buffer at 2 at first
+- Set the audio **Interface** to the one you want (there may be several in the list).
+- Set the **Sample Rate to 48000**.
+- Set the **Frames/Period to 128** and Periods/Buffer at 2 at first.
 
-Restart JACK for the new settings to take effect
+Restart JACK for the new settings to take effect.
+
+### Configure PipeWire
+
+The Jamulus Client requires a JACK sound server (native PipeWire support has not yet been implemented).
+PipeWire provides a compatibility layer that emulates the JACK server, so when you launch the Jamulus Client, however PîpeWire automatically exposes itself to it as a JACK server.
```

Comment:
This sentence got a bit muddled up - "however" should be moved to the start or removed altogether.